modules/adopter-registration-impl/src/main/java/org/opencastproject/adopter/statistic/ScheduledDataCollector.java

```diff
@@ -199,6 +247,42 @@ private String collectStatisticData(String statisticKey) throws Exception {
 
     statisticData.setSeriesCount(seriesService.getSeriesCount());
     statisticData.setUserCount(userAndRoleProvider.countAllUsers());
+
+    SearchQuery sq = new SearchQuery();
+    sq.withId("");
+    sq.withElementTags(new String[0]);
+    sq.withElementFlavors(new MediaPackageElementFlavor[0]);
+    sq.signURLs(false);
+    sq.includeEpisodes(true);
+    sq.includeSeries(false);
+
+    List<Organization> orgs = organizationDirectoryService.getOrganizations();
+    statisticData.setTenantCount(orgs.size());
+
+    for (Organization org : orgs) {
+      SecurityUtil.runAs(securityService, org, systemAdminUser, () -> {
+        //Calculate the number of attached CAs for this org, add it to the total
+        long current = statisticData.getCACount();
+        int orgCAs = caStateService.getKnownAgents(systemAdminUser, org).size();
+        statisticData.setCACount(current + orgCAs);
+
+        //Calculate the total number of minutes for this org, add it to the total
+        current = statisticData.getTotalMinutes();
+        long orgDuration = 0L;
+        try {
+          SearchResult sr = searchService.getForAdministrativeRead(sq);
+          orgDuration = Arrays.stream(sr.getItems())
```

Comment:
First, this doesn't work iirc since the search service automatically applies a limit to queries by default. Though maybe the `getForAdministrativeRead` bypasses that.

If it does get all, this will load hundreds of thousands of Java objects into memory. I don't think that's a good idea. This may cause harm to the node.

Reply:
Ah, good point.  Luckily, the first page of results appears to contain a correct `total` value.

Reply:
Per discussion in chat, this is disabled completely for now and will make a reappearance later.